tensorflow/compiler/xla/service/hlo_parser.cc

```diff
@@ -636,7 +636,7 @@ bool HloParserImpl::ParseInstructionList(HloComputation** computation,
     // the pool, which should not happen.
     if (root_node == nullptr) {
       LOG(FATAL) << "instruction " << root_name
-                 << " was marked as ROOT but the parser has not seen it before";
+                 << " was marked as ROOT but the parser has not seen it before"; // LOG(FATAL) crashes the program by calling abort()
```

Comment:
Final comment: please end the sentence with period.

Reply:
Added. Please check now. Thanks.